package.json

```diff
@@ -54,5 +54,14 @@
     "tsconfig-paths": "^3.9.0",
     "typedoc": "^0.23.21",
     "typescript": "^4.9.3"
+  },
+  "dependencies": {
+    "@fast-check/jest": "^1.7.2",
+    "@matrixai/async-init": "^1.9.1",
+    "@matrixai/contexts": "^1.2.0",
+    "@matrixai/id": "^3.3.6",
```

Comment:
Do you actually use this dependency? And `ix` too?

Reply:
Uh, async-init? or contexts?


Its for the new Create Destroy (or rather just destroy)

Reply:
Do you use ix?

Reply:
no

Reply:
Then why do you have it?